app/services/requirement_form_json_service.rb

```diff
@@ -624,6 +624,10 @@ def formio_type_options
                "energy_step_code_report_file"
              )
             file_hash[:tooltip] = ENERGY_STEP_CODE_TOOLTIP_URL
+          elsif input_options["computed_compliance"].present?
```

Comment:
Based on my analysis of the code, the elsif condition on line 627 will never be reached. Here's why:
Both the if and elsif conditions check for variants of "computed compliance," but the if condition is more specific.
The if condition on line 623 specifically checks if the requirement key ends with "energy_step_code_report_file". This is a special case of a requirement with automated compliance.
The elsif condition on line 627 checks for a more general case: input_options["computed_compliance"].present?. This key is present for any requirement that has automated compliance, including the "energy step code report file".